Managers/SettingsMan.cpp

```diff
@@ -9,13 +9,23 @@
 #include "NetworkClient.h"
 #include "NetworkServer.h"
 
+#ifdef __unix__
+#include "System/Config.h"
+#endif
+
 namespace RTE {
 
 	const std::string SettingsMan::c_ClassName = "SettingsMan";
 
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 	void SettingsMan::Clear() {
+#if !LINUX_PORTABLE
```

Comment:
This should be the other way around? i.e if portable build then `"Settings.ini"` and if not then `"Base.rte/Settings.ini"`.
Or am I understanding it wrong?
Also use `ifdef` and `ifndef`.

Reply:
Portable build wouldn't be installed so cc can write to Base.rte/Settings.ini -> This is the correct way around